app/api/auth/login/individual/route.ts

```diff
@@ -0,0 +1,45 @@
+import { ConflictError } from "@/custom/errors/dictionary/errorDictionary";
+import { handleErrorEdgeCases } from "@/custom/errors/handler/errorHandler";
+import { connectMongoDB } from "@/lib/mongo_connect/mongoConnect";
+import {
+  AccountIndividual,
+  type IndividualSchemaTypes,
+} from "@/models/auth/IndividualSchema";
+import bcrypt from "bcrypt";
+import { NextResponse as res } from "next/server";
+
+export async function POST(request: Request) {
+  try {
+    await connectMongoDB();
+
+    const data = await request.json();
+
+    const { email, password } = data;
+
+    const user = await AccountIndividual.findOne<IndividualSchemaTypes>(
```

Comment:
Please re-format this line to be:
const user = await AccountGallery.findOne({email}, "user_id, password"})

This is the recommendation from the mongoose documentation